prometheus/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM prom/prometheus:v2.10.0
+
+ADD ./prometheus.yml /etc/prometheus/prometheus.yml
+
```

Comment:
Prometheus needs a configuration file to gather data.

At that file we will tell prometheus where he must scrape data and how often he will scrape it.

This configuration file can be found [here](https://github.com/okfn-brasil/serenata-de-amor/pull/484/files#diff-30171a37a35ae9a4f53c346d781da3efR1)